main.cpp

```diff
@@ -126,7 +136,138 @@ i cubed: 531441
 Use a service like https://www.diffchecker.com/diff to compare your output. 
 */
 
-#include <iostream>
+
+template<class NumType>
+struct Numeric
+{
+    //using Type = Temporary<NumType>;
+    using Type = NumType;
+
+    Numeric(Type nuNum) : value( std::make_unique<Type>(nuNum) ) { } 
+
+    operator Type() const {return *value;} //possibly this line
```

Comment:
If your class owns a `Temporary<NumType>` and your conversion function is meant to return a `NumType`, then you need to convert your `Temporary` into `NumType` using the tools you learned in Project 4.3
- [ ] revise